package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "new-project",
+  "name": "Pathfinder",
```

Comment:
I *think* it has to be all lower case

Reply:
Oh I see! Okay, I'll get that fixed. Thank you!

Reply:
It should be fixed now.